check_all.sh

```diff
@@ -5,6 +5,6 @@ echo "Begin check..." \
 && python gen_pyi.py \
 && black -l 100 -t py36 --pyi --fast matplotlib-stubs/pyplot.pyi.in \
 && flake8 *-stubs \
-&& python -m pytest -vv tests/ \
-&& mypy tests \
+&& python -m pytest -vv tests/py37+ \
```

Comment:
btw, this is where I was thinking of using the `python -V` to have a conditional about which tests to run/check based on the python version.